Index: fusioninventory-agent/lib/FusionInventory/Agent/Tools/Generic.pm
===================================================================
--- fusioninventory-agent.orig/lib/FusionInventory/Agent/Tools/Generic.pm
+++ fusioninventory-agent/lib/FusionInventory/Agent/Tools/Generic.pm
@@ -286,7 +286,7 @@ sub _loadPCIDatabase {
     my (%params) = @_;
 
     ($PCIVendors, $PCIClasses) = _loadDatabase(
-        file => "$params{datadir}/pci.ids"
+        file => "/usr/share/misc/pci.ids"
     );
 }
 
